src/Fixer/LanguageConstruct/ClassKeywordRemoveFixer.php

```diff
@@ -159,7 +159,7 @@ private function replaceClassKeyword(Tokens $tokens, $classIndex)
         $classBeginIndex = $classEndIndex;
         while ($tokens[--$classBeginIndex]->isGivenKind(array(T_NS_SEPARATOR, T_STRING)));
         ++$classBeginIndex;
-        $classString = $tokens->generatePartialCode($classBeginIndex, $classEndIndex);
+        $classString = ltrim($tokens->generatePartialCode($classBeginIndex, $classEndIndex), '\\');
```

Comment:
from what i shortly checked recently we don't handle comments in `::class` construction anyway in this fixer
different bug, still nice to fix